lib/Command/Configure.php

```diff
@@ -313,6 +314,29 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 				$providerConfig->setUser($username);
 				$providerConfig->setApiKey($apiKey);
 
+				break;
+			case 'plivo':
+				$config->setProvider($provider);
+				/** @var PlivoConfig $providerConfig */
+				$providerConfig = $config->getProvider()->getConfig();
+				
+				$authIdQuestion = new Question('Please enter your plivo authentication id (Auth ID): ');
+				$authId = $helper->ask($input, $output, $authIdQuestion);
+				
+				$authTokenQuestion = new Question('Please enter your plivo authentication token (Auth Token): ');
+				$authToken = $helper->ask($input, $output, $authTokenQuestion);
+				
+				$srcNumberQuestion = new Question('Please enter your plivo phone number (in E.164 format): ');
```

Comment:
I would recommend to supply example, because E.164 is not widely known.

Reply:
copy that boss